sds/src/scanner/regex_rule/mod.rs

```diff
@@ -0,0 +1,29 @@
+pub mod compiled;
+pub mod config;
+//
```

Comment:
Would it be ok to remove this comment?

Reply:
done https://github.com/DataDog/dd-sensitive-data-scanner/pull/85/commits/ffc0fad9d7bd59165d2b75ad63d31a4f69429827